Interfaces/GCHP/gchp_chunk_mod.F90

```diff
@@ -1504,16 +1505,6 @@ SUBROUTINE GCHP_Chunk_Run( GC,                                             &
        _ASSERT(RC==GC_SUCCESS, 'Error calling Set_AerMass_Diagnostic')
     ENDIF
 
-    CALL MAPL_TimerOff( STATE, 'GC_DIAGN' )
-    if(Input_Opt%AmIRoot.and.NCALLS<10) write(*,*) ' --- Diagnostics done!'
-
-    !=======================================================================
-    ! Convert State_Chm%Species units
-    !=======================================================================
-    CALL Convert_Spc_Units ( Input_Opt, State_Chm, State_Grid, State_Met, &
-                             OrigUnit, RC )
-    _ASSERT(RC==GC_SUCCESS, 'Error calling CONVERT_SPC_UNITS')
```

Comment:
Could you explain what this change is? Is it supposed to be outside of an `#ifdef ADJOINT` block?

Reply:
See next comment.